.swiftlint.yml

```diff
@@ -4,22 +4,12 @@ disabled_rules: # rule identifiers to exclude from running
   - nesting
   - operator_whitespace
   - trailing_whitespace
-  - type_body_length
-  - variable_name
+included: # paths to include during linting. `--path` is ignored if present.
+  - Sources
```

Comment:
When running `danger-swiftlint` on Travis, it'll check everything including the Fastlane files that have been downloaded to run within the VM. This forces only the `Sources` folder to be linted. So it's future-proofing.